scheduler/reconcile.go

```diff
@@ -95,6 +95,10 @@ type reconcileResults struct {
 	// stop is the set of allocations to stop
 	stop []allocStopResult
 
+	// annotationUpdates are updates to the allocation that are not from a
+	// jobspec change.
+	annotationUpdates map[string]*structs.Allocation
```

Comment:
Better name suggestion - `internalStateUpdates` or `attributeUpdates` -  these are changes to some attributes or internal state of the annotation